tests/CocktailParty.tex

```diff
@@ -65,21 +64,25 @@ \subsection{Additional rules and remarks}
 
 	\item \textbf{Changing places:} After giving the order (when the robot is not in the party room), the referees may re-arrange the people including their body posture. That is, a sitting person may change to a standing posture and vice versa.
 
-	\item \textbf{Positions and orientations:} All people roughly stay where they are (if not asked to move by the referees), but they are allowed to move in certain limits (e.g. turn around, make a step aside). They do not need to look at the robot, but are requested to do so, when instructed by the robot.
+	\item \textbf{Positions and orientations:} All test participants roughly stay where they are (if not asked to move by the referees), but they are allowed to move in certain limits (e.g. turn around, make a step aside). They do not need to look at the robot, but are requested to do so, when instructed by the robot.
 
-	\item \textbf{Empty arena:} During the test, only the robot, the guest, and the Barman are in the arena. The door opener, the referees and other personnel that is not assigned as test people will be outside the scenario.
+	\item \textbf{Empty arena:} During the test, only the robot, the guest, and the Bartender are in the arena. The door opener, the referees and other personnel that is not assigned as test people will be outside the scenario.
 
-	\item \textbf{Calling instruction:} The team needs to specify before the test which ways of getting the attention of the robot are allowed. This can be waving, calling or both of them. The robot can also decide to skip this part, by asking for people to get close to it.
+	\item \textbf{Calling instruction:} The team needs to specify before the test which ways of getting the attention of the robot are allowed for standing persons. This can be waving, calling or both of them. The robot can also decide to skip this part, by asking for people to get close to it.
```

Comment:
> ```This can be waving, calling or both of them.```

1. Other ways of calling are allowed (e.g. whistling), so maybe it's worth rephrase to make clear that waving and calling are not the only accepted ways of getting the robot's attention. Teams can bring their own alternatives. TC might even accept a team giving smart buttons to each person.
2. I think there is a comma missing after ```calling```.

Reply:
Here you are asking me to introduce a change of the rules.
Such things should be discussed in another PR.
I think it isn't a good idea to allow teams the introduction of artificial ways of calling a waiter.

Reply:
Skip it then